kernel/common/ezautolinkoperator.php

```diff
@@ -60,6 +91,7 @@ function modify( $tpl, $operatorName, $operatorParameters, $rootNamespace, $curr
     {
         $ini = $tpl->ini();
         $max = $ini->variable( 'AutoLinkOperator', 'MaxCharacters' );
+        $simpleWww = ( $ini->variable( 'AutoLinkOperator', 'ChangeUrlWithoutProtocol' ) == 'enabled' );
```

Comment:
We think that this var name should be more clear and specific on what it represents. While perhaps too long even $linkWwwUrlWithoutProtocol or $linkWwwUrl would be better